import_ldraw.py

```diff
@@ -204,7 +211,7 @@ def parse_line(self, line):
         color = line[1]
 
         if color == '16':
-            color = self.colour
+            color = self.color
```

Comment:
OK, so you changed all instances of colour. Normally I would save such cleanup for a non-feature PR, but I'll let it slide this time. I assume you tested everything and it all still works?